bff/solid/src/view/timebar_section.tsx

```diff
@@ -155,11 +155,11 @@ export function TimebarSection(props: ITimebarSectionProps) {
 function TimeframeSelector() {
   const { setState } = contextState();
   const id = "timeframeSelector";
-  const options = [
+  const optionsTimespan = [
     { ms: 14 * 24 * 60 * 60 * 1000, label: "last 14d", ms2: 60 * 60 * 1000 },
-    { ms: 7 * 24 * 60 * 60 * 1000, label: "last 7d", ms2: 30 * 60 * 1000 },
-    { ms: 2 * 24 * 60 * 60 * 1000, label: "last 2d", ms2: 30 * 60 * 1000 },
-    { ms: 1 * 24 * 60 * 60 * 1000, label: "last 1d", ms2: 30 * 60 * 1000 },
+    // { ms: 7 * 24 * 60 * 60 * 1000, label: "last 7d", ms2: 30 * 60 * 1000 },
+    // { ms: 2 * 24 * 60 * 60 * 1000, label: "last 2d", ms2: 30 * 60 * 1000 },
+    // { ms: 1 * 24 * 60 * 60 * 1000, label: "last 1d", ms2: 30 * 60 * 1000 },
```

Comment:
As far as I remember, we decided not to disable these. Let's revert this change.

Reply:
It's my understanding that they are supposed to be removed. We can always add them back in on Monday.